SCYLLA-VERSION-GEN

```diff
@@ -0,0 +1,49 @@
+#!/bin/sh
+
+PRODUCT=scylla
+VERSION=666.development
+
+if test -f version
```

Comment:
Why is this file part of "enable integration tests"?

Reply:
it's a mistke

Reply:
move into the correct commits